src/pages/developers/local-environment.tsx

```diff
@@ -0,0 +1,314 @@
+import { GetStaticProps } from "next"
+import { StaticImageData } from "next/image"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { useTranslation } from "react-i18next"
+import { Box, Flex, ListItem, SimpleGrid, UnorderedList } from "@chakra-ui/react"
+
+import { ChildOnlyProp } from "@/lib/types"
+
+import FeedbackCard from "@/components/FeedbackCard"
+import { Image } from "@/components/Image"
+import Heading from "@/components/OldHeading"
+import Text from "@/components/OldText"
+import PageMetadata from "@/components/PageMetadata"
+import ProductCard from "@/components/ProductCard"
+import Translation from "@/components/Translation"
+
+import { getRequiredNamespacesForPage } from "@/lib/utils/translations"
+
+import { ghRepoData } from "@/lib/api/ghRepoData"
+import EthDiamondBlackImage from '@/public/assets/eth-diamond-black.png'
+import EpirusImage from '@/public/dev-tools/epirus.png'
+import FoundryImage from '@/public/dev-tools/foundry.png'
+import HardhatImage from '@/public/dev-tools/hardhat.png'
+import KurtosisImage from '@/public/dev-tools/kurtosis.png'
+import ScaffoldEthImage from '@/public/dev-tools/scaffoldeth.png'
+import TruffleImage from '@/public/dev-tools/truffle.png'
+import WaffleImage from '@/public/dev-tools/waffle.png'
+import EthBlocksImage from '@/public/developers-eth-blocks.png'
+
+const Content = ({ children }: ChildOnlyProp) => {
+  return (
+    <Box py={4} px={8} w="full">
+      {children}
+    </Box>
+  )
+}
+
+const Column = ({ children }: ChildOnlyProp) => {
+  return (
+    <Box
+      flex="1 0 33%"
+      justifyContent="flex-end"
+      mb={6}
+      mr={8}
+      w="full"
+      maxW={{ base: "full", md: "none" }}
+    >
+      {children}
+    </Box>
+  )
+}
+
+interface IFramework {
+  id: string
+  url: string
+  githubUrl: string
+  background: string
+  name: string
+  description: string
+  alt: string
+  image: StaticImageData
+  starCount?: number
+  languages?: string[]
+}
+
+export const getStaticProps: GetStaticProps = async (
+  context
+) => {
+  const { locale } = context
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage('/developers/local-environment')
+
+  // TODO: fetch github repo data
+  let frameworksList: Array<IFramework> = [
```

Comment:
We could place this array outside of the `getStaticProps` hook to improve readability.